src/fragments/gen2/quickstart/build-a-backend.mdx

```diff
@@ -50,7 +50,7 @@ WORK IN THE FRONTEND CODE FILE.)
 
 Using JavaScript or Next.js React Server Components, Middleware, Server
 Actions, or Pages Router? Review how to generate Data clients for those use
-cases: https://docs.amplify.aws/gen2/build-a-backend/data/connect-to-API/
+cases: https://docs.amplify.aws/javascript/build-a-backend/data/connect-to-API/
```

Comment:
For these links should we default to javascript or nextjs?

Reply:
react is our default

Reply:
Changed it to react